Gemfile

```diff
@@ -54,7 +54,6 @@ group :gcs do
 end
 
 group :gae do
-  gem 'rubyzip'
 end
```

Comment:
If the non-standard library is no longer required, I think it is better to drop this group altogether.